docs/v0.7/en/contributor-guide/datanode/metric-engine.md

```diff
@@ -0,0 +1,39 @@
+# Metric Engine
+
+## Overview
+
+The `Metric` engine is a component of GreptimeDB, and it's an implementation of the storage engine. It mainly targets scenarios with a large number of small tables for observable metrics.
+
+Its main feature is to use synthetic physical wide tables to store a large amount of small table data, achieving effects such as reuse of the same column and metadata. This reduces storage overhead for small tables and improves columnar compression efficiency. The concept of a table becomes even more lightweight under the `Metric` engine.
+
+## Concepts
+
+The `Metric` engine introduces two new concepts: "logical table" and "physical table". From the user's perspective, logical tables are exactly like ordinary ones. From a storage point-of-view, physical Regions are just regular Regions.
```

Comment:
```suggestion
The `Metric` engine introduces two new concepts: "logical table" and "physical table". From the user's perspective, logical tables are exactly like ordinary ones.
```

We talk about table. Why suddenly "physical Regions". Is there "logical regions"?